### ISSUE TITLE
cant run on RPi2

### ISSUE THREAD

**Issue by rodcar125** [projectweekend/Pi-GPIO-Server#2]
Hi!

After install i just cant run it on my RPi2. When I start the program with python from the command line y get the following...

pi@S3RPi ~/Pi-GPIO-Server $ python pi_gpio_server.py 
Traceback (most recent call last):
  File "pi_gpio_server.py", line 1, in <module>
    from pi_gpio import app, socketio
  File "/home/pi/Pi-GPIO-Server/pi_gpio/**init**.py", line 16, in <module>
    import urls
  File "/home/pi/Pi-GPIO-Server/pi_gpio/urls.py", line 4, in <module>
    from handlers import PinList, PinDetail
  File "/home/pi/Pi-GPIO-Server/pi_gpio/handlers.py", line 3, in <module>
    from config.pins import PinHttpManager
  File "/home/pi/Pi-GPIO-Server/pi_gpio/config/pins.py", line 2, in <module>
    from .gpio import BaseGPIO
  File "/home/pi/Pi-GPIO-Server/pi_gpio/config/gpio.py", line 1, in <module>
    import RPi.GPIO as GPIO
RuntimeError: This module can only be run on a Raspberry Pi!

Is this normal due to lack of support for RpPi2 or did I messed up something im not aware of?

Thanks and great job!
Greetings,
Rodrigo C

**Comment by projectweekend** [comment]
Thanks for the feedback! I only recently got a Pi2 so I haven't been able to test yet. That said, I did a little research on your error and the problem is in the RPi.GPIO library. That library needs an upgrade on the Pi2 before it can work. Check this out article: [https://blog.adafruit.com/2015/02/05/how-to-fix-error-loading-rpi-gpio-python-library-on-your-brand-new-raspberry-pi-2/](https://blog.adafruit.com/2015/02/05/how-to-fix-error-loading-rpi-gpio-python-library-on-your-brand-new-raspberry-pi-2/). I will put a fix in the install script in next couple days so that this library gets upgraded as mentioned in this article. In the meantime, feel free to update it manually.
